README.md

```diff
@@ -152,4 +152,13 @@ You can also find other bindings on [GitHub](https://github.com/search?q=tantivy
 - and [more](https://github.com/search?q=tantivy)!
 
 ### On average, how much faster is Tantivy compared to Lucene?
-- According to our [search latency benchmark](https://tantivy-search.github.io/bench/), Tantivy is approximately 2x faster than Lucene.
\ No newline at end of file
+- According to our [search latency benchmark](https://tantivy-search.github.io/bench/), Tantivy is approximately 2x faster than Lucene.
+
+### Does tantivy support incremental indexing?
+- Yes.
+
+### How can I edit documents?
+- Data in tantivy is immutable. To edit a document, the document needs to be deleted and reindexed.
+
+### When will my documents be searchable during indexing?
+- Documents will be searchable after a `commit`. A commit can be executed manually, or when reaching the memory budget during indexing. Comitting means the segment will be flushed to disk. 
```

Comment:
That's incorrect. 

A commit can only be done manually.

Segment getting serialized to disk can happen when we reach the memory budget, but the data will not be made searchable until a manual commit is done.

It might be important to also add that existing reader's need to be reload for changes to be visible.

Reply:
Documents will be searchable after a `commit`, which can only be done manually. When reaching the memory budget during indexing, a segment can get serialized to disk, but the data will not be searchable until a commit is done (not necessary to repeat "manual commit" here imo). It is important to reload the existing reader for changes to be visible.